### PR TITLE
Add section 1 content

### PR DESCRIPTION
**Pull request by BonnieJRobert**
Please review new content in `materials\01-what-is-vc-qmd`

### CODE REVIEW

**Comment by BonnieJRobert** [comment]
@lindsay-fredrick this branch was created prior to your section 2 review in Teams.  So, none of those changes have been incorporated in this branch.

File: materials/01-what-is-vc.qmd [hunk 1]
```diff
@@ -2,12 +2,63 @@
...
 ---
 
-## here's some code to copy
+## 1.1 What is Version Control?
 
-```{r}
-#| echo: true
-#| code-copy: true
-rnorm(10)
+Version control, also known as source control, is the practice of tracking and managing changes to software code. Version control systems are software tools that help software teams manage changes to source code over time.
```
Comment (lindsay-fredrick) [inline materials/01-what-is-vc.qmd:7]: I might mention you don't have to specifically be a software team (as most BC stats'ers are not). Just anyone with any type of code/documents that you want under VC. Most useful for code for sure but has other uses.

File: materials/01-what-is-vc.qmd [hunk 2]
```diff
@@ -2,12 +2,63 @@
...
+## 1.3 What is a Version Control Repository?
+
+In version control systems, a repository is a data structure that stores metadata for a set of files or directory structure. Some of the metadata that a repository contains includes, among other things, a historical record of changes in the repository, a set of commit objects, and a set of references to commit objects, called heads. We will go into more detail on these during the workshop.
+
+Essentially, a version control repository keeps track of a set of files in a project. Using the WES analysis example above, you and your colleagues may choose to structure your version control project like this:
+
+```         
+├──wes-analysis
+|  ├── data
+|  │   ├── wes-results.csv
```
Comment (lindsay-fredrick) [inline materials/01-what-is-vc.qmd:50]:
```suggestion
|  │   ├── wes-results
```

File: materials/01-what-is-vc.qmd [hunk 3]
```diff
@@ -2,12 +2,63 @@
...
+|  ├── ref
+|  │   ├── references
+|  │   ├── images
+|  ├── plots
+|  ├── code
+|  ├── .git
 ```
 
-## 
+Which is essentially a regular project folder containing subfolders to organize your work, but with an additional `.git` folder in which to house the metadata.
```
Comment (lindsay-fredrick) [inline materials/01-what-is-vc.qmd:64]: I would add a disclaimer or something here - this makes it sound like they have to manually create the `.git` folder themselves, whereas that will be handled by either git or GitHub, depending on how you set it up.

**Review by lindsay-fredrick — APPROVED** [review]
Looks good to me.  If there's time, one thing we might want to add here is success stories from within BC Stats - projects that have used/currently use VC in some fashion to make their lives easier (people always want examples of past work).

Some ideas might be:
- Using GitHub to share code to more quickly produce some NLP code for the demographic survey.
- Using GitHub to store code for the Small Business Dashboard that will need to be updated annually
- Metadata viewer
- Using GitLab in the SRE to sync and collaborate across teams/projects

**Pull request merged**